Code/Framework/AzCore/AzCore/Math/Vector4.h

```diff
@@ -8,10 +8,15 @@
 
 #pragma once
 
-#include <AzCore/Math/Vector3.h>
+#include <AzCore/Math/Internal/MathTypes.h>
+#include <AzCore/RTTI/TypeInfoSimple.h>
 
 namespace AZ
 {
+    class Vector3;
+    class Vector2;
+    class ReflectContext;
```

Comment:
The forward declaration of ReflectContext isn't needed

Reply:
there is a static method that requires ReflectContext.
```
static void Reflect(ReflectContext* context);
```

Reply:
Oh, I missed that. Conversation Resolved.